File: ios/Sources/ScrollbarHiderPlugin/ScrollbarHider.swift
import Foundation
import WebKit

@objc public class ScrollbarHider: NSObject {
    @objc public func hideScrollbars(webView: WKWebView) {
        print("ScrollbarHider: Hiding scrollbars on WKWebView")
        webView.scrollView.showsVerticalScrollIndicator = false
        webView.scrollView.showsHorizontalScrollIndicator = false
    }
}

File: android/src/main/java/com/owaish/plugins/scrollbarhider/ScrollbarHider.java
package com.owaish.plugins.scrollbarhider;

import android.util.Log;
import android.webkit.WebView;

public class ScrollbarHider {

    public void hideScrollbars(WebView webView) {
        Log.i("ScrollbarHider", "Hiding scrollbars on WebView");
        if (webView != null) {
            webView.setVerticalScrollBarEnabled(false);
            webView.setHorizontalScrollBarEnabled(false);
        }
    }
}
